examples/dataflow-hashpipeline/scripts/seed-firestore.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud import firestore
+from hashlib import blake2b
+from base64 import b64decode
+import os
+import random
+import sys
+
+os.environ["GCLOUD_PROJECT"] = sys.argv[1]
+SALT = sys.argv[2]
+with open("scripts/key.b64") as f:
+  KEY = b64decode(f.read())
+
+def random_ssn():
+  # Get a random formatted SSN
+  area = f'{random.randrange(1, 799):03d}'
+
+  # SSNs cannot start with 666...because Satan
```

Comment:
there's actually a lot more of these rules such as 000 
You should see suggestion to use Faker which has more [test coverage](https://github.com/joke2k/faker/blob/master/tests/providers/test_ssn.py) for this domain
ssa.gov/kc/SSAFactSheet--IssuingSSNs.pdf